index.php

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+        <title>Lexi Slome</title>
+        <link rel="stylesheet" href="index-style.css?Time=<?php echo microtime()?>" />
+    </head>
+
+    <body>
+        <ul>
+            <li><a href='http://local.codingfellowship.com:8083/about_me.html'>About Me</a></li>
+            <li><a href='http://local.codingfellowship.com:8083/running.php'>running</a></li>
+            <li><a href='http://local.codingfellowship.com:8083/reading.php'>reading</a></li>
+            <li><a href='http://local.codingfellowship.com:8083/crafting.php'>crafting</a></li>
+            <li><a href='http://local.codingfellowship.com:8083/CookingAndBaking.php'>cooking and baking</a></li>
+            <li><a href='http://local.codingfellowship.com:8083/exploring.php'>exploring</a></li>
+        </ul>
+
+        <br>
+        <br>
+
+        <h1>Hi, welcome to my website!</h1>
+    <div class="ProfilePic">
+        <img src='https://scontent.xx.fbcdn.net/v/t31.0-8/14054591_1096149677131041_8531721397845641299_o.jpg?oh=7c719fdcc961495eb364e6f24e623b92&oe=59ABE08D' alt='profile picture'>
```

Comment:
It works to include images from external sites like this, but it's not a great idea. For starters, Facebook might not like being your image host (their servers get hit everytime someone loads this image). Also, if this image changes or stops working, it'll break your website.

A better approach is to download the image from Facebook and add it to your project (I'd make an "images" folder). That way your site is hosting all of the images you use.